test/sharness/lib/test-lib.sh

```diff
@@ -312,3 +312,33 @@ test_str_contains() {
 	shift
 	echo "$@" | grep "$find" >/dev/null
 }
+
+test_links_in_trickle_dag() {
+	local LINKS="$(ipfs object links $1)"
+	local i="0"
```

Comment:
not sure but i think `local` is a bashism. we're restricted to Bourne Shell (`sh`) for portability.